marklogic-data-hub/src/test/ml-modules/root/test/data-hub-test-helper.sjs

```diff
@@ -250,5 +254,6 @@ module.exports = {
   getStagingRecord,
   stagingDocumentExists,
   verifyJson,
+  getUrisInCollection,
```

Comment:
Same thing, use alphabetical order (though I don't mind the amped functions being at the end since they're special)